components/04-indicators/alert/alert.hbs

```diff
@@ -1,3 +1,4 @@
 <div class="alert {{ modifier }}" role="alert">
+  {{> '@font-icon' }}
```

Comment:
Ik wil niet dat de implementator de vrijheid heeft een ander icoon te kunnen selecteren. Daarbij is dit een breaking change in het alert component geworden, wat volgens mij niet nodig is.

We kunnen dmv. mixins het icoon vanuit de styling plaatsen. Dan wisselen we van svgbackgroundimage naar webfont, maar nog steeds vanuit styling

Reply:
Je hebt het `<i>` element toch nodig voor het icoon en het cirkeltje? 

Het juiste icoon wordt nu wel herleid uit de alert-class.